src/lhttpc.erl

```diff
@@ -32,18 +32,22 @@
 -module(lhttpc).
 -behaviour(application).
 
--export([start/0, stop/0, request/4, request/5, request/6, request/9]).
--export([start/2, stop/1]).
--export([
-        send_body_part/2,
-        send_body_part/3,
-        send_trailers/2,
-        send_trailers/3
-    ]).
--export([
-        get_body_part/1,
-        get_body_part/2
-        ]).
+%% this is the API module; xref ignore all exports
+-ignore_xref(
+   [start/0, stop/0, start/2, stop/1
+    , lb_status/0
+    , request/4, request/5, request/6, request/9
+    , send_body_part/2, send_body_part/3
+    , send_trailers/2, send_trailers/3
+    , get_body_part/1, get_body_part/2]).
+
+-export(
+   [start/0, stop/0, start/2, stop/1
+    , lb_status/0
+    , request/4, request/5, request/6, request/9
+    , send_body_part/2, send_body_part/3
+    , send_trailers/2, send_trailers/3
+    , get_body_part/1, get_body_part/2]).
```

Comment:
I try to keep the existing style (whatever that is) in pull requests. But in this case there was multiple competing styles.
Would you prefer to keep the multiple styles or to pick one of them?